src/main/java/com/example/sharemind/admin/application/AdminServiceImpl.java

```diff
@@ -86,6 +87,7 @@ public void updateProfileStatus(Long counselorId, Boolean isPassed) {
             profileStatus = ProfileStatus.EVALUATION_FAIL;
         }
         counselor.updateProfileStatus(profileStatus);
+        counselor.updateProfileUpdatedAt();
```

Comment:
profileUpdatedAt을 만들면서 프로필 상태가 바뀔때마다 갱신을 해줬는데 혹시 다른 로직에서 인증 실패인데 프로필 시간을 업데이트 해주면 생길 수 있는 문제가 있을까요?
검색쪽에서는 애초에 인증된 상담사만 보여주어 문제가 있을 것 같지는 않습니다!

Reply:
당장은 떠오르지 않네요...! 하지만 나중에 그런 상황이 발생할 수도 있으니 아예 하나로 묶어버리는 건 어떨까요?
```suggestion
        counselor.updateProfileStatusAndProfileUpdatedAt(profileStatus);
```